fhir-ig-davinci-pdex-plan-net/src/main/java/com/ibm/fhir/ig/davinci/pdex/plan/net/NetResourceProvider.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.ig.davinci.pdex.plan.net;
+
+import com.ibm.fhir.registry.util.PackageRegistryResourceProvider;
+
+public class NetResourceProvider extends PackageRegistryResourceProvider {
```

Comment:
pretty minor, but i'd have prefered `PlanNetResourceProvider`

Reply:
I'm happy to acquiesce to your naming preferences.